test/SILOptimizer/diagnostic_constant_propagation_floats_x86.swift

```diff
@@ -1,7 +1,7 @@
 // RUN: %target-swift-frontend -emit-sil -primary-file %s -o /dev/null -verify
 //
 // REQUIRES: CPU=i386 || CPU=x86_64
-// UNSUPPORTED: OS=windows-msvc
+// UNSUPPORTED: OS=windows-msvc, OS=linux-android
```

Comment:
I think that this should be marked as `UNSUPPORTED` on both Windows and Android with a note that they do not support FP80. Would you mind adding that note?

Reply:
No problem, will do.

Reply:
Added the note